File: web_service_ITDEL/app/Models/Kaos.php
<?php

namespace App\Models;

use Illuminate\Database\Eloquent\Factories\HasFactory;
use Illuminate\Database\Eloquent\Model;

class Kaos extends Model
{
    use HasFactory;
    protected $table = 'kaos';
    protected $fillable = [
        'ukuran', 'harga',
    ];

    public function pemesanan()
    {
        return $this->hasMany(Pemesanan::class);
    }
}

File: it_del/lib/Models/kaos.dart
class Kaos {
  int? id;
  String? ukuran;
  int? harga;

  Kaos({
    this.id,
    this.ukuran,
    this.harga,
  });

  factory Kaos.fromJson(Map<String, dynamic> json) {
    return Kaos(
      id: json['id'] as int?,
      ukuran: json['ukuran'] as String?,
      harga: json['harga'] as int?,
    );
  }
}
